easybuild/tools/options.py

```diff
@@ -392,6 +392,7 @@ def override_options(self):
             'debug-lmod': ("Run Lmod modules tool commands in debug module", None, 'store_true', False),
             'default-opt-level': ("Specify default optimisation level", 'choice', 'store', DEFAULT_OPT_LEVEL,
                                   Compiler.COMPILER_OPT_FLAGS),
+            'default-debug-symbols': ("Sets default value of debug toolchain option", None, 'store_true', True),
```

Comment:
Maybe `retain-debug-symbols-by-default` is a better name for this configuration option?

Enabling it by default makes sense I think, there *should* be no fallout of that...

Reply:
Well I won't insist, though I find the wording of "retain" a bit strange, reads to me that we were intentionally stripping them out.

Reply:
As agreed in sync call:
```suggestion
            'keep-debug-symbols': ("Sets default value of debug toolchain option", None, 'store_true', True),
```